client/post-editor/editor-mobile-navigation/index.jsx

```diff
@@ -23,17 +57,26 @@ const EditorMobileNavigation = React.createClass( {
 
 		return (
 			<div className="editor-mobile-navigation">
-				<Site indicator={ false } site={ this.props.site } />
-				<button className="button editor-mobile-navigation__toggle" onClick={ this.toggleSidebar }>
-					{ this.translate( 'Actions' ) }
-				</button>
-				<Gridicon
-					icon="cross"
-					onClick={ this.props.onClose }
-					className="editor-mobile-navigation__close" />
+				<div className="editor-mobile-navigation__actions">
+					<Gridicon
+						icon="chevron-left"
+						onClick={ this.props.onClose }
+						className="editor-mobile-navigation__icon separator-right" />
+					<Gridicon
+						icon="pencil"
+						onClick={ this.closeSidebar }
+						className={ classnames( 'editor-mobile-navigation__icon', {
+							'is-selected': ! this.state.sidebarOpen
+						} ) } />
+					<Gridicon
+						icon="cog"
+						onClick={ this.openSidebar }
+						className={ classnames( 'editor-mobile-navigation__icon', 'separator-right', {
```

Comment:
Might consider using `:last-of-type` or `:last-child` pseudo-class than an explicit class.